Update the antivirus docs to mention that continue is for debug purposes only (#547)
owncloud/docs-ocis: modules/ROOT/pages/deployment/services/s-list/antivirus.adoc

  -   *delete*: (default): Infected files will be deleted immediately, further post-processing is cancelled.
  -   *abort*:  (advanced option): Infected files will be kept, further post-processing is cancelled. Files can be manually retrieved and inspected by an admin. To identify the file for further investigation, the antivirus service logs the abort/infected state including the file ID. The file is located in the `storage/users/uploads` folder of the ocis data directory and persists until it is manually deleted by the admin via the xref:{s-path}/storage-users.adoc#manage-unfinished-uploads[Manage Unfinished Uploads] command.

>   -   *continue*:  (debug option): Infected files will be marked via metadata as infected but post-processing continues normally. Note: Infected Files are moved to their final destination and therefore not prevented from download which includes the risk of spreading viruses.

Reviewer: ```suggestion
  -   *continue*:  (not recommended - only for debugging): Infected files will be marked via metadata as infected but post-processing continues normally. Note: Infected Files are moved to their final destination and therefore not prevented from download which includes the risk of spreading viruses.
```

Author: yes that makes it clearer, updated
